### DISCUSSION TITLE
Git repo stats not showing for public repo

### DISCUSSION THREAD

**Discussion by normandy7** [squidfunk/mkdocs-material#5468] [Q&A] [answered]
We've added a URL to our (public) repository, but for some reason the tag, stars, and forks are missing.

Is there anything apart from repo privacy that could be messing with displaying the stats?

Link to our site: https://docs.neptune.ai/

**Comment by squidfunk** [comment, marked as answer]
Make sure to clear your browser cache (you can just close an open it again), there might be stale data. It works for me:

![screenshot-docs-neptune-ai-1683210468041](https://user-images.githubusercontent.com/932156/236238285-da42b244-8c02-42a2-bf2f-15f4e458badd.png)

**Reply by normandy7** [reply]
I wrongly assumed a hard reload would be enough; clearing the site data worked. Thanks!
